docs/get-started.md

```diff
@@ -26,6 +26,8 @@ If you have not signed up before, you can do so [here](https://improbable.io/get
     - **Linux Build Support**
     - **Mac Build Support**
 
+> **Note:** You need Linux build support because the cloud servers run on Linux. <br/>When you come to build your project, in the `Assets/Fps/Config/BuildConfiguration`, do not change the `UnityGameLogic Cloud Environment` from Linux.
```

Comment:
because the cloud servers run on Linux.
->
because all server-workers are running in a Linux environment when running a cloud deployment

something like that? just making it very clear that this only applies to server-workers